types/src/transaction/script.rs

```diff
@@ -107,11 +107,11 @@ impl fmt::Debug for Script {
 /// Call a Move entry function.
 #[derive(Clone, Debug, Hash, Eq, PartialEq, Serialize, Deserialize)]
 pub struct EntryFunction {
-    module: ModuleId,
-    function: Identifier,
-    ty_args: Vec<TypeTag>,
+    pub module: ModuleId,
+    pub function: Identifier,
+    pub ty_args: Vec<TypeTag>,
     #[serde(with = "vec_bytes")]
-    args: Vec<Vec<u8>>,
+    pub args: Vec<Vec<u8>>,
```

Comment:
why when we have getters below?

Reply:
Needed for conversion logic which creates this struct. I can add a constructor but that doesn't seem better than making these public, at least in my noob Rust experience?

Reply:
Well the data structure went from immutable to mutable. So this is definitely not a desirable pattern.

Reply:
Fixed